src/hydro_system.hpp

```diff
@@ -470,11 +493,20 @@ void HydroSystem<problem_t>::ComputeFlatteningCoefficients(amrex::MultiFab const
 
 		if constexpr (reconstruct_eint) {
 			// compute (rho e) (gamma - 1)
-			Pplus2 *= primVar(i + 2, j, k, primDensity_index) * (gamma_ - 1.0);
-			Pplus1 *= primVar(i + 1, j, k, primDensity_index) * (gamma_ - 1.0);
-			P *= primVar(i, j, k, primDensity_index) * (gamma_ - 1.0);
-			Pminus1 *= primVar(i - 1, j, k, primDensity_index) * (gamma_ - 1.0);
-			Pminus2 *= primVar(i - 2, j, k, primDensity_index) * (gamma_ - 1.0);
+			amrex::GpuArray<Real, nmscalars_> massScalars = RadSystem<problem_t>::ComputeMassScalars(primVar, i + 2, j, k);
```

Comment:
Can you rename `massScalars` as appropriate for each index, e.g. `massScalars_plus2`, `massScalars_minus2`, etc.?